docs/UserGuide.md

```diff
@@ -376,7 +462,9 @@ Example:
 * `customise show a/` The application now shows addresses in the list of contacts.
 * `customise show p/ t/` The application now shows phone numbers and tags in the list of contacts.
 
-## Todo Management Features
```

Comment:
Isnt todo features separated from contact features?

Reply:
Yes. That's why it's deleted?